### ISSUE TITLE
Create a CNN Using Transfer Learning

### ISSUE THREAD

**Issue by ierolsen** [ierolsen/Udacity-Dog-Breed-Classifier#4]
## **Model Architecture**
The submission specifies a model architecture that uses part of a pre-trained model.

## **Question 5:** Model Architecture
The submission details why the chosen architecture is suitable for this classification task.

## **Train and Validate the Model**
Train your model for a number of epochs and save the result with the lowest validation loss.

## **Test the Model**
Accuracy on the test set is 60% or greater.

## **Predict Dog Breed with the Model**
The submission includes a function that takes a file path to an image as input and returns the dog breed that is predicted by the CNN.
